docs/src/pages/components/avatar.mdx

```diff
@@ -55,5 +59,11 @@ The `Avatar` component renders a generic avatar graphic, or a stylized label.
             <td>string</td>
             <td>-</td>
         </tr>
+        <tr>
+            <td>alt</td>
+            <td>Replace the default avatar alt tag when a `src` is provided.</td>
```

Comment:
Backticks don't work inside the table, have to use `pre` tags.

Reply:
👍